google/cloud/grpc_options.h

```diff
@@ -96,7 +96,7 @@ struct GrpcTracingOptionsOption {
 /**
  * The size of the background thread pool
  *
- * @note this is ingored if `GrpcBackgroundThreadsFactoryOption` is set.
+ * @note This is ingored if `GrpcBackgroundThreadsFactoryOption` is set.
```

Comment:
Might as well `s/ingored/ignored/` while we're here.

Reply:
Done.